deployment/k8s/charts/components/bulkLoad/values.yaml

```diff
@@ -1,3 +1,5 @@
+sharedLogsVolumeEnabled: false
+sharedLogsPvc: ""
```

Comment:
hmmm - this isn't very obvious (to clear the pvc) - what happens if this isn't done?